.dockerignore

```diff
@@ -1,6 +1,5 @@
 **/.classpath
 **/.dockerignore
-**/.env
```

Comment:
This is standard `.dockerignore`.

You can use `.env.development` and `.env.production` files

Reply:
I removed it because I hadn't added region specific .env files because I don't know what the intention is for having different regions. .env isn't even used in any other part of the project, so really not relevant

Reply:
OK, let's live with it right now. But I will prefer to comment that line:

````
#**/.env # Disabled in PR #44
```